src/kepler_model/estimate/estimator.py

```diff
@@ -160,15 +161,22 @@ def sig_handler(signum, frame) -> None:
     default="info",
     required=False,
 )
-def run(log_level: str):
+@click.option(
+    "--machine-spec",
+    type=str,
```

Comment:
this can be a path  see usage: https://github.com/sustainable-computing-io/kepler/blob/main/e2e/tools/validator/src/validator/cli/__init__.py#L243 

and click will check if the path exists etc ..